web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -251,38 +251,38 @@ class CVEAudit extends React.Component<Props, State> {
                     </div>
                     <div id={`collapse-${status}`} className="panel-collapse collapse in">
                       <div className="panel-body">
-                        The client is affected by a vulnerability that received a patch, but Uyuni is unable to locate any of the patches in
-                        relevant channels.
+                        The client is affected by a vulnerability that received a patch, but Uyuni is unable to locate
```

Comment:
This message should be translated using the `t()` function

Reply:
Yep, this was just for the demo, the message also needs to vary between patch statuses. Right now, it shows the same description for all of them.

Reply:
Done. I'm assuming the generation of the translations will be done automatically.